.ci/Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env groovy
 
-@Library('apm@current') _
+@Library('apm@test/node') _
```

Comment:
This is just for testing purposes, so the nvm/npm/nodejs installation that happens on the fly will report back what some details about what it has been installed. See https://github.com/elastic/apm-pipeline-library/pull/1928/files#diff-afc97124fbb38137b00b422a1f1b72d6a458857071329f80456684ee4891d280

Reply:
```suggestion
@Library('apm@current') _
```